docs/logs/correlation/README.md

```diff
@@ -0,0 +1,63 @@
+# Logs correlation
+
+The getting started docs for [logs](../getting-started/README.md) and
+[traces](../../traces/getting-started/README.md) showed how to emit logs and
+traces independently, and export them to console exporter.
+
+This doc explains how logs can be correlated to traces.
+
+## Logging Data Model support for correlation
+
+[Logging Data
+Model](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#trace-context-fields)
+defines fields which allow a log to be correlated with span (`Activity` in
+.NET). The fields `TraceId` and `SpanId` allows a log to be correlated to
+corresponding `Activity`.
+
+## Correlation in OpenTelemetry .NET
+
+The good news is that, in OpenTelemetry .NET SDK, there is no user action
+required to enable correlation. i.e the SDK automatically enables logs to
+`Activity` correlation, by populating the fields `TraceId` and `SpanId` from the
```

Comment:
I think we also have:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/641b2f76f1e3b9b1aa1697ab47ec24aed31a45c3/src/OpenTelemetry/Logs/LogRecord.cs#L74-L76